websites/S/SuperiorServers.co/presence.ts

```diff
@@ -0,0 +1,177 @@
+      } else if (document.location.pathname.includes("/milrp")){
+        presenceData.state = "(MilRP)";
+      } else if (document.location.pathname.includes("/cwrp")){
+        presenceData.state = "(CWRP)";
+      }
+    }
+    if (document.location.pathname.includes("/leaderboard/money")){
+      presenceData.details = "Viewing money leaderboards";
+      presenceData.startTimestamp = currentTime;
+      if (document.location.pathname.includes("/darkrp")){
+        presenceData.state = "(DarkRP)";
+      }
+    }
+    if (document.location.pathname.includes("/moneyboard")){
+      presenceData.details = "Viewing money leaderboards";
+      presenceData.startTimestamp = currentTime;
+      if (document.location.pathname.includes("/cwrp")){
+        presenceData.state = "(CWRP)";
+      } else if (document.location.pathname.includes("/milrp")){
+        presenceData.state = "(MilRP)";
+      }
+    }
+    if (document.location.pathname.includes("/darkrp/leaderboard/orgs")){
+      presenceData.details = "Viewing org leaderboard";
+      presenceData.state = "(DarkRP)";
+      presenceData.startTimestamp = currentTime;
+    }
+    if (document.location.pathname.includes("/polls")){
+      presenceData.details = "Viewing polls";
+      presenceData.startTimestamp = currentTime;
+      if (document.location.pathname.includes("/darkrp")){
+        presenceData.state = "(DarkRP)";
+      } else if (document.location.pathname.includes("/milrp")){
+        presenceData.state = "(MilRP)";
+      } else if (document.location.pathname.includes("/cwrp")){
+        presenceData.state = "(CWRP)";
+      }
+    }
+    if (document.location.pathname.includes("/characters")){
+      presenceData.details = "Using character search";
+      presenceData.startTimestamp = currentTime;
+      if (document.location.pathname.includes("/darkrp")){
+        presenceData.state = "(DarkRP)";
+      } else if (document.location.pathname.includes("/milrp")){
+        presenceData.state = "(MilRP)";
+      } else if (document.location.pathname.includes("/cwrp")){
+        presenceData.state = "(CWRP)";
+      }
+    }
+    if (document.location.pathname.includes("/gamemasters")){
+      presenceData.startTimestamp = currentTime;
+      if (document.location.pathname.includes("/darkrp")){
+        presenceData.state = "(DarkRP)";
+      } else if (document.location.pathname.includes("/milrp")){
+        presenceData.details = "Viewing MilRP GMs";
+        let GMs = document.querySelector("#gamemasters_info").textContent;
+        GMs = GMs.substr((GMs.indexOf(" of") + 3), GMs.indexOf(" entries")).replace(" entries", "");
+        presenceData.state = GMs + " total members";
+      } else if (document.location.pathname.includes("/cwrp")){
+        presenceData.details = "Viewing CWRP GMs";
+        let GMs = document.querySelector("#gamemasters_info").textContent;
+        GMs = GMs.substr((GMs.indexOf(" of") + 3), GMs.indexOf(" entries")).replace(" entries", "");
```

Comment:
please change it to const and do substr after textContent because you can do it obviously